scripts/core/EXT_EventQueryKernelTimestamps.rst

```diff
@@ -74,27 +74,27 @@ This extension enables the querying of synchronized event timestamps.
     // synchronize host
     // ...
 
-    if (supportsRaw && supportsSynchronized) {
+    if (supportsKernel && supportsSynchronized) {
         // Number of event timestamps
         uint32_t count = 0;
 
         // Get the number of timestamps associated with the event.
         ${x}EventQueryKernelTimestampsExt(hEvent, hDevice, &count, nullptr);
 
-        // Allocate storage for raw device timestamps
-        std::vector<${x}_kernel_timestamp_result_t> rawTimestamps(count);
+        // Allocate storage for kernel timestamp results
+        std::vector<${x}_kernel_timestamp_result_t> timestamps(count);
 
-        // Allocate storage for synchronized event timestamps, in nanoseconds
+        // Allocate storage for synchronized timestamp results, in nanoseconds
```

Comment:
type for synchronized timestamps should also be `kernel_timestamps_result_t`?

Reply:
@aravindksg 
Not sure whether "kernel_timestamps_result_t" would be appropriate here, since spec mentions the below for "ze_kernel_timestamp_data_t"

_The timestamp frequency can be queried from ::ze_device_properties_t.timerResolution.
The number of valid bits in the timestamp value can be queried from ::ze_device_properties_t.kernelTimestampValidBits._

This may not be true for the synchronized timestamp.
We may have to create a new structure, unless we have something that represents host.

Reply:
@joshuaranjan 
Would it be sufficient to add a note to the details:
```
details:
    - "The timestamp frequency can be queried from $x_device_properties_t.timerResolution."
    - "The number of valid bits in the timestamp value can be queried from $x_device_properties_t.kernelTimestampValidBits."
    - "1.6": "**Note**: The `timerResolution` and `kernelTimestampValidBits` members of `$x_device_properties_t` does not apply to synchronized timestamp data"
```

Reply:
@joshuaranjan good point. should below `std::vector<uint64_t>` be `std::vector<std::chrono::nanoseconds> synchronizedTimestamps(count)` ? @wdamon-intel

Reply:
apologies, please disregard above comment, seeing that in latest commit this was fixed

Reply:
@wdamon-intel 

> @joshuaranjan Would it be sufficient to add a note to the details:
> 
> ```
> details:
>     - "The timestamp frequency can be queried from $x_device_properties_t.timerResolution."
>     - "The number of valid bits in the timestamp value can be queried from $x_device_properties_t.kernelTimestampValidBits."
>     - "1.6": "**Note**: The `timerResolution` and `kernelTimestampValidBits` members of `$x_device_properties_t` does not apply to synchronized timestamp data"
> ```

Yes.
Also the spec mentions
_uint64_t kernelStart [out] **device clock** at start of kernel execution_

Here device clock might not apply as well.

So to handle both , if we can have something like the below, would it be OK ?
- "1.6": "**Note**: For synchronized timestamp data, the `timerResolution` and `kernelTimestampValidBits` members of `$x_device_properties_t` does not apply. Also the kernel timestamps are host timestamps

@aravindksg please share your thoughts

Reply:
@joshuaranjan, @aravindksg 
Instead of all these notes, I'm thinking it'll be better to introduce new data structures: `ze_synchronized_timestamp_data_ext_t` and `ze_synchronized_timestamp_result_ext_t`